app/helpers/url_helper.rb

```diff
@@ -39,6 +39,10 @@ def artwork_url(artwork_id)
     "#{Convection.config.artsy_url}/artwork/#{artwork_id}"
   end
 
+  def my_collection_artwork_url(artwork_id)
+    "#{Convection.config.artsy_url}/my-collection/artworks/#{artwork_id}/edit"
```

Comment:
I was surprised to see the plural here, given that the artwork views use `.../artwork/*`, but apparently it's correct. I think we should make them consistent, but that's separate.